bgppeer/bgppeer.go

```diff
@@ -32,11 +34,14 @@ func (b *BGPPeer) ListBGPPeers() (*calicoApi.BGPPeerList, error) {
 }
 
 func (b *BGPPeer) SaveBGPPeer(peer *calicoApi.BGPPeer) error {
-	if peer.GetUID() == "" {
+	exists, _ := b.CalicoClient.BGPPeers().Get(context.Background(), peer.Name, options.GetOptions{})
```

Comment:
This step is wasting resources, would be nice to avoid fetching configs one by one.

Reply:
Removed, thank you!